docs/ODESwagger.yaml

```diff
@@ -713,10 +713,10 @@ definitions:
         description: IPv4 address of RSU
       rsuUsername:
         type: string
-        description: SNMP username
+        description: SNMP username (Not required if value is set in application.properties or .env file)
```

Comment:
The user doesn't know or care about inner-workings of ODE. Please change to `(Not required if value is pre-configured in ODE properties)`